src/spanningTree/spanningTree_driver.cpp

```
@@ -0,0 +1,264 @@
+        hint = edges_sql;
+        edges = get_edges(edges_sql, normal, false);
+        hint = "";
+
+        if (edges.empty()) {
+            notice << "No edges found";
+            log << edges_sql;
+
+            if (which == BFS) return;
+
+            auto emptyresults = pgrouting::only_root_result(roots);
+            return_count = get_tuples(emptyresults, return_tuples);
```

Cppcheck (2.19.0)

[style] The function 'edge_not_relaxed' is never used.

(unusedFunction)